lib/RoadizRozierBundle/templates/custom-forms/actionsMenu.html.twig

```diff
@@ -1,14 +1,24 @@
-<nav data-action-menu class="actions-menu uk-navbar">
-    {% if (is_granted('ROLE_ACCESS_CUSTOMFORMS_DELETE')) %}
-        {% if item.id %}
-            <a class="uk-button uk-button-small rz-no-ajax-link" href="{{ path('customFormsDuplicatePage', { id: item.id}) }}">
-                <span class="icon-container"><i class="uk-icon-rz-duplicate"></i></span>
-                <span class="label"><span class="label-text">{% trans %}customForm.duplicate{% endtrans %}</span></span>
-            </a>
-            <a class="uk-button uk-button-danger" href="{{ path('customFormsDeletePage', { id: item.id}) }}">
-                <span class="icon-container"><i class="uk-icon-rz-trash-o"></i></span>
-                <span class="label"><span class="label-text">{% trans %}delete{% endtrans %}</span></span>
-            </a>
+
+{% embed "@RoadizRozier/macros/rz_floating_bar.html.twig" with {
+    vertical: true,
+    class: 'actions-menu'
+} %}
+    {% block items %}
+        {% if (is_granted('ROLE_ACCESS_CUSTOMFORMS_DELETE')) %}
+            {% if item.id %}
+                {{ _self.item(
+                    label: 'customForm.duplicate',
+                    class: 'rz-button--tertiary',
+                    href: path('customFormsDuplicatePage', { id: item.id}) ,
+                    icon: 'rz-icon-ri--file-copy-line'
+                ) }}
+                {{ _self.item(
+                    label: 'delete',
+                    class: 'rz-button--error',
+                    href: path('customFormsDeletePage', { id: item.id}) ,
+                    icon: 'rz-icon-ri--delete-bin-7-line'
+                ) }}
```

Comment:
@eliot488995568 @manuelodelain 

Why not create a specialized item only for delete buttons `_self.deleteItem()` ?
You are duplicating all code except for `path('customFormsDeletePage', { id: item.id})`. This way, if we decide to change icon, all delete buttons will use the same.